lesson_2/cmd/app/main.go

```diff
@@ -0,0 +1,18 @@
+package main
+
+import (
+	"../../pkg/fib"
+	"fmt"
+)
+
+func main() {
+	var n int
+
+	defer fmt.Println("Complete!")
+	fmt.Print("This program can count fibonacci numbers\n", "Please enter number: ")
+	fmt.Scan(&n)
+
+	fib.NumL(n) //by loop
+	fib.NumR(n)	//by recursion
```

Comment:
get here results from functions and then print it. It helps you to use package "fmt" only in one place(module) of your project